projects/common/src/utilities/formatters/date/date-formatter.ts

```diff
@@ -72,11 +76,23 @@ export class DateFormatter {
       return '-';
     }
 
-    return formatDate(coercedValue, this.getFormatString(), 'en_US');
+    return formatDate(coercedValue, this.getFormatString(coercedValue), 'en_US');
   }
 
-  private getFormatString(): string {
+  private getFormatString(date: Date): string {
     switch (this.options.mode) {
+      case DateFormatMode.TimeWithTimeZoneOffset:
+        /*
+         * WARNING: This format is currently unsupported by at least some backend APIs e.g. Reporting
+         */
+        const timezone = date.getTimezoneOffset();
+        const sign = timezone <= 0 ? '+' : '-';
+
+        const pad = (n: number): string => `${n < 10 ? '0' : ''}${n}`;
```

Comment:
This still shouldn't be needed. Angular format definitely supports ISO, right?

Reply:
Ah, my mistake. I was looking at the native JS time formatting options. Angular expands what is available. Updated. Thanks!